src/app/components/dashboard-company-cart/dashboard-company-cart.coffee

```diff
@@ -0,0 +1,29 @@
+
+#============================================
+#
+#============================================
+DashboardCompanyCartCtrl = ($scope, $state, $stateParams, $uibModal, MnoeCurrentUser, MnoeOrganizations, MnoeProvisioning, MnoeConfig) ->
+  'ngInject'
+
+  cartSubscriptions = ->
+    params = { where: {staged_subscriptions: true } }
```

Comment:
Isn't is possible to keep the JSON API format? You could then also remove this filter: 
```ruby
params = { where: { status: 'staged' } }
```
You could then also remove this filter: https://github.com/maestrano/maestrano-hub/pull/991/files#diff-c8fbb65db37a91bbb5be88241f81e874R34. It seems that everywhere you use it, you can just say `where(status: 'staged')`